infra/utils.py

```diff
@@ -25,7 +25,8 @@
 
 ALLOWED_FUZZ_TARGET_EXTENSIONS = ['', '.exe']
 FUZZ_TARGET_SEARCH_STRING = 'LLVMFuzzerTestOneInput'
-VALID_TARGET_NAME = re.compile(r'^[a-zA-Z0-9_-]+$')
+VALID_TARGET_NAME_REGEX = re.compile(r'^[a-zA-Z0-9_-]+$')
+BLOCKLISTED_TARGET_NAME_REGEX = re.compile(r'^(jazzer_driver.*)$')
```

Comment:
very small nit but can you call this DENYLISTED or DISALLOWED?

Reply:
This is taken [verbatim from ClusterFuzz](https://github.com/google/clusterfuzz/blob/f0855c662ed07f2ee01fa93cc8fe579a10a13076/src/clusterfuzz/_internal/bot/fuzzers/utils.py#L29), as is most of this method as far as I can tell. Should I still make the change?

Reply:
Oh, it's okay it just seemed like a small typo for blacklisted. We'll keep it as is, thanks for the reference. If there's cleanup they can both be cleaned up together.